frontend/src/concepts/modelServing/ModelStatusIcon.tsx

```
@@ -101,17 +103,26 @@ export const ModelStatusIcon: React.FC<ModelStatusIconProps> = ({
-        data-testid="model-status-text"
-        style={{ width: 'fit-content', cursor: 'pointer' }}
-        onClick={onClick}
-      >
-        {statusSettings.label}
-      </Label>
+      {hideLabel ? (
+        <Icon
+          style={{ cursor: 'pointer' }}
+          status={statusSettings.status}
+          color={statusSettings.color}
+          children={statusSettings.icon}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)